src/icons/Help.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import Svg, { Circle, Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: colors
+}
+
+const Help = ({ color = colors.gray3 }: Props) => (
```

Comment:
argh, i strongly feel like we don't need to duplicate this Help icon just because of the props, i know you said you got build issues so i'll try from my side

Reply:
does this work for you? https://github.com/valora-inc/wallet/pull/3493